plugins/tiddlywiki/multiwikiserver/modules/store/sql-tiddler-database.js

```diff
@@ -12,1437 +12,1476 @@ Validation is for the most part left to the caller
 
 (function() {
 
-/*
-Create a tiddler store. Options include:
-
-databasePath - path to the database file (can be ":memory:" to get a temporary database)
-engine - wasm | better
-*/
-function SqlTiddlerDatabase(options) {
-	options = options || {};
-	const SqlEngine = require("$:/plugins/tiddlywiki/multiwikiserver/store/sql-engine.js").SqlEngine;
-	this.engine = new SqlEngine({
-		databasePath: options.databasePath,
-		engine: options.engine
-	});
-	this.entityTypeToTableMap = {
-		bag: {
-			table: "bags",
-			column: "bag_name"
-		},
-		recipe: {
-			table: "recipes",
-			column: "recipe_name"
+	/*
```

Comment:
The diffs here are messed up by the formatting changes which makes it hard to review them, so I'll come back to this once the formatting is fixed.